ethcore/private-tx/src/encryptor.rs

```diff
@@ -111,11 +111,11 @@ impl SecretStoreEncryptor {
 			return Ok(key);
 		}
 		let contract_address_signature = self.sign_contract_address(contract_address)?;
-		let requester = self.config.key_server_account.ok_or_else(|| ErrorKind::KeyServerAccountNotSet)?;
+		let requester = self.config.key_server_account.ok_or_else(|| Error::KeyServerAccountNotSet)?;
```

Comment:
Unrelated to this PR but this is a needless closure should be `ok_or` instead `Error::KeyServerAccountNotSet` is just value